### PR TITLE
Tomster filter

### PR DESCRIPTION
**Pull request by ultimatemonty**
This popped up this evening when I was showing my daughter all the different Zoeys and I thought "man it'd be nice to be able to see just the Zoeys".

Let there be filtering!

Caveat: I know **nothing** about Ruby/Middleman and figured this out mostly by Googling so I likely did some ignorant things. Please point these things out so I can make them mo betta!

### CODE REVIEW

**Comment by wifelette** [comment]
Pulled this down and took a look, this is great! 

I can't decide if I think the category titles should be pluralized, a la Tomsters/Zoeys/Meetups/Conferences/etc. But I _think_ I think that? Thoughts, @ultimatemonty?

**Comment by ultimatemonty** [comment]
@wifelette I'm good either way :)

Did the ruby/Middleman side of things looks good? One thing that was kind of bothering me was the dynamic URLs end in a `/` and it seems like they shouldn't. Probably an easy way to fix that but I wasn't able to find anything on the Googles

Also, sorry for dropping a surprise PR on you! I realized after the fact (and @acorncom kindly pointed it out on Slack) that opening an issue before PR'ing is the right approach for this kind of stuff. Will make sure to do so in the future!

**Comment by wifelette** [comment]
Having slept on it, let's pluralize :p Can you make the tweak?

RE: Middleman/Ruby: truthfully while I use it a lot, my level of expertise isn't really... existing ;) I've learned to do the very basics of what I need and not much more than that.

Indeed it could probably be prettier—your work _and_ the work done prior—but lacking an available expert I say we ship it as is and maybe someone else can tidy it later if indeed there's a better way. Or if anyone reading this _is_ said expert, by all means, please jump in!

**Comment by ultimatemonty** [comment]
@wifelette will do later this evening!

File: source/tomster.html.erb [hunk 1]
```diff
@@ -1,20 +1,29 @@
 ---
 title: "Tomster and Zoey"
+filters: all, tomster, zoey
 ---
 
 <div class="tomster section intro">
   <h1>Tomster and Zoey</h1>
   <p>
-    Tomster was the first of many friendly faces of the Ember project and community. Over the years we've dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you'll see Ember friends around even more.
+    Tomster was the first of many friendly faces of the Ember project and community. Over the years we"ve dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you"ll see Ember friends around even more.
```
Comment (acorncom) [inline source/tomster.html.erb]: The quote here changed from a single to a double quote, could you change that back?

File: source/tomster.html.erb [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
 </div>
 
 <div class="tomster section list">
+  <div id="tomster-filter">
+    <%= active_link_to "All", "/tomster/" %> |
+    <%= active_link_to "Tomster", "/tomster/tomster/" %> |
+    <%= active_link_to "Zoey", "/tomster/zoey/" %> |
+    <%= active_link_to "Meetup", "/tomster/meetup/" %> |
+    <%= active_link_to "Conference", "/tomster/conference/" %> |
+    <%= active_link_to "Other", "/tomster/other/" %>
```
Comment (acorncom) [inline source/tomster.html.erb]: Let's change the above links to `/tomster?filter=tomster` and equivalent  Doing that will make life simpler if/when we move this page to Ember (as we don't have to worry about having additional pages to map on the search engine side of things).

**Edit**: hmm, looks like that's not an option we can accommodate with Middleman, carry on ... ;-)

**Comment by ultimatemonty** [comment]
@acorncom @locks @wifelette I believe I've addressed all of your feedback

* Filter now works via queryParams instead of dynamic pages (for future compat)
* Use flexbox for layout as opposed to tables
* Fixed copy pasta errors

**Comment by ultimatemonty** [comment]
@locks commits squashed

**Comment by locks** [comment]
Thank you for your contribution @ultimatemonty 😁  Keep'em coming!

**Comment by acorncom** [comment]
Very snazzy, thanks @ultimatemonty 👍

**Pull request merged**